articles/modules/ROOT/pages/what-is-pattern-comprehension-and-why-should-i-care.adoc

```diff
@@ -0,0 +1,78 @@
+= What is pattern comprehension and why should I care?
+:slug: what-is-pattern-comprehension-and-why-should-i-care
+:author: Sven
+:category: cypher
+:tags: cypher, OGM, SDN, Mapping
+:neo4j-versions: 3.5, 4.x
```

Comment:
Interestingly enough, pattern comprehensions date back to Neo4j 3.1! You may need to add all the relevant minor versions here.